src/crimson/common/gated.h

```diff
@@ -52,4 +68,43 @@ class Gated {
   seastar::gate pending_dispatch;
 };
 
-}// namespace crimson::common
+class gate_per_shard : public Gated {
```

Comment:
The inheritance relationship doesn't look valid. No logic/member seems to be inherited.

Reply:
yes, will remove that, after i removed the close, there is no need in that inheritance